GVFS/GVFS.Build/GVFS.props

```
@@ -3,7 +3,7 @@
 
   <PropertyGroup Label="Parameters">
     <GVFSVersion>0.2.173.2</GVFSVersion>
-    <GitPackageVersion>2.20190312.1</GitPackageVersion>
+    <GitPackageVersion>2.20190513.1-pr</GitPackageVersion>

```

```suggestion
    <GitPackageVersion>2.20190521.1-pr</GitPackageVersion>
```

(provided that [the build](https://dev.azure.com/gvfs/ci/_build/results?buildId=11485) succeeds...)

---

The build succeeded ;-)

---

I applied the suggestion from my phone.

---

I applied the suggestion from my phone.

---

Thanks, @dscho! That worked!